### PR TITLE
QF-2208-Content_sweep_employer_web

### PR DESCRIPTION
**Pull request by karen-hanna**

### CODE REVIEW

File: src/SFA.DAS.EmployerRequestApprenticeTraining.Web/Views/EmployerRequest/Dashboard.cshtml [hunk 1]
```diff
@@ -129,10 +129,8 @@
                         else
                         {
                             <p class="govuk-body">
-                                Requests expire after @Model.Dashboard.RemovedAfterExpiryMonths months and are no longer seen by training providers.
-                            </p>
-                            <p class="govuk-body">
-                                When your requests have expired, you'll see them here
+                                Requests expire after @Model.Dashboard.RemovedAfterExpiryMonths months and are no longer seen by training providers.
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.EmployerRequestApprenticeTraining.Web/Views/EmployerRequest/Dashboard.cshtml:132]: In the story it is ... training providers. When you requests....

File: src/SFA.DAS.EmployerRequestApprenticeTraining.Web/Views/EmployerRequest/Dashboard.cshtml [hunk 2]
```diff
@@ -118,9 +118,9 @@
                                     </div>
                                 </p>
                                 <p>
-                                    <div class="govuk-body govuk-!-margin-bottom-1">Request made on @request.RequestedAt.ToString("d MMM yyyy")</div>
+                                    <div class="govuk-body govuk-!-margin-bottom-1">Request made on @request.RequestedAt.ToString("d MMMM yyyy")</div>
```
Comment (ChrisJWoodcock) [inline src/SFA.DAS.EmployerRequestApprenticeTraining.Web/Views/EmployerRequest/Dashboard.cshtml:121]: Even though the story doesn't say I think it's that all uses of MMM need to be converted to MMMM - like in other pages - but confirm that

**Review by ChrisJWoodcock — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.EmployerRequestApprenticeTraining.Web/Views/EmployerRequest/Dashboard.cshtml (hunk 1 above)
Comment (karen-hanna) [reply src/SFA.DAS.EmployerRequestApprenticeTraining.Web/Views/EmployerRequest/Dashboard.cshtml:132]: According to AC 6, the expired tab should look like this when thre are no expired requests 
image

Comment (ChrisJWoodcock) [reply src/SFA.DAS.EmployerRequestApprenticeTraining.Web/Views/EmployerRequest/Dashboard.cshtml:132]: Yes so I was questioning the carriage return introduced from removed the p tag

**Review by ChrisJWoodcock — APPROVED** [review]

**Pull request merged**